i18n/en.json

```diff
@@ -320,6 +320,26 @@
     "translation": "%s updated the channel header to: %s"
   },
   {
+    "id": "api.channel.post_update_channel_purpose_message_and_forget.join_leave.error",
+    "translation": "Failed to post join/leave message %v"
```

Comment:
Yeah, that message isn't right. This should also be renamed since the "join_leave" part of the name doesn't make sense here